### ISSUE TITLE
Performance in Cora

### ISSUE THREAD

**Issue by zhuo931077127** [danielegrattarola/keras-gat#30]
Hey, I run your code without modifying anything, but obtaining bad performances under default setting(Cora dataset).

> Epoch 99/10000
1/1 [==============================] - 0s 78ms/step - loss: 0.1020 - acc: 0.0569 - val_loss: 0.3604 - val_acc: 0.0395y with val_weighted_acc available, skipping.
Epoch 100/10000
1/1 [==============================] - 0s 80ms/step - loss: 0.1021 - acc: 0.0480 - val_loss: 0.3605 - val_acc: 0.0343y with val_weighted_acc available, skipping.
Epoch 101/10000
1/1 [==============================] - 0s 74ms/step - loss: 0.1021 - acc: 0.0679 - val_loss: 0.3605 - val_acc: 0.0273y with val_weighted_acc available, skipping.
Epoch 102/10000
1/1 [==============================] - 0s 82ms/step - loss: 0.1018 - acc: 0.0731 - val_loss: 0.3605 - val_acc: 0.0218y with val_weighted_acc available, skipping.
Done.
Test loss: 0.7192684412002563
Test accuracy: 0.046159528195858

My environment is TensorFlow2.3-gpu and Keras 2.4.3.

**Comment by danielegrattarola** [comment]
Hey, 
sorry, this is an old implementation based on TF 1.15 and Keras < 2.3.
Many things changed between TF1 and TF2, and the results may not be reproducible using different versions (I know, weird, but you can try for yourself). 

I have added a deprecation warning on the README, and I suggest you look at the up-to-date, more modern [Spektral](https://graphneural.network) library which also features a [GAT example](https://github.com/danielegrattarola/spektral/blob/master/examples/node_prediction/citation_gat.py). 

Cheers

**Comment by zhuo931077127** [comment]
Thanks for your nice work. I have done some simple modifications on this code and obtain expected results.
